DigitalLearningSolutions.Web/Styles/index.scss

```diff
@@ -131,7 +131,7 @@ h1#page-heading {
 
 .nhsuk-summary-list__value {
   @include govuk-media-query($until: tablet) {
-    word-break: break-word;
+    word-break: break-all;

```

Comment:
Doesn't break-all break the second word instead of putting it on a new line when there's two words in one line?

Example here:
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break